components/google-cloud/google_cloud_pipeline_components/aiplatform/batch_predict_job/component.yaml

```diff
@@ -236,7 +236,7 @@ implementation:
           ', "input_config": {',
           '"instances_format": "', {inputValue: instances_format}, '"',
           ', "gcs_source": {',
-          '"uris":',  {inputValue: gcs_source_uris},
+          '"uris": "',  {inputValue: gcs_source_uris}, '"',
```

Comment:
Thanks for the PR. 
the gcs_source_uris is JsonArray and not suppose to be wrapped by double quotes. Could you confirm the current code is indeed causing issues?

Reply:
hi @IronPan - yes - in researching this more closely I found that the value was indeed an array.  You can ignore or decline this PR - sorry for the noise.

Reply:
@IronPan how are we supposed to pass these values?

When passing this through as a pipeline param typed to `List[str]` it fails to compile with this error:
```
kfp.dsl.types.InconsistentTypeException: Incompatible argument passed to the input "gcs_source_uris" of component "model_batch_predict": Argument type "typing.List[str]" is incompatible with the input type "JsonArray"
```

```python
@kfp.v2.dsl.pipeline(
    name="batch-predict-v0",
    pipeline_root=pipeline_gcs_root_path)
def batch_predict_pipeline(
                           ...
                           gcs_source_uris: List[str],
                           ...
                           ):


        predict_op = ModelBatchPredictOp(
            ...
            gcs_source_uris=gcs_source_uris,
            ....
        )

```